elteikthesis.cls

```diff
@@ -247,7 +256,6 @@
 
 % ------------------------------
 % Helyközök és tördelés
-\linespread{1.5}
```

Comment:
Is there an additional benefit of setting the line spread through the `setspace` package? If not, the simpler version should be kept.

Reply:
Yes, `setspace` doesn't change the spacing in 'qeustionable' like the footnotes, it also keeps single spacing in code listings, which is very ugly with 1.5 spacing imo.

Reply:
I agree that the 1.5x line spacing should not be applied on code listings and mathematical formulas.
Rules for footnotes are not really unified in this matter in my experience. E.g. the *Chicago Manual of Style* (which is one of the most well known formatting guidelines) states that text should be double-spaced, but footnotes *can* be single-spaced.